### PR TITLE
closes #729

### LINKED ISSUE #729: Results page of HHpred is still broken
ss_dssp line is now empty; see production server for comparison.

hhpred

Q ss_pred, T ss_dssp, T ss_pred could also be missing and we shouldn't show empty lines. Previous logic dealt with all of this.

### PR DESCRIPTION
**Pull request by vikramalva**

### CODE REVIEW

**Comment by zy4** [comment]
haha, nice timing :)

**Comment by vikramalva** [comment]
What a coincidence! I another small issue as well. :)

File: modules/results/src/main/twirl/views/hhpred/hit.scala.html [hunk 1]
```diff
@@ -85,14 +86,15 @@
...
-    @if(!hit.template.ss_pred.isEmpty) {
+    @if(!hit.template.ss_dssp.isEmpty) {
         @{
-            Html(Common.makeRow("sequence", Array("", "T ss_pred", "", Common.SSColorReplace(hit.template.ss_pred))))
+            Html(Common.makeRow("sequence", Array("", "T ss_dssp", "", Common.SSColorReplace(hit.template.ss_dssp))))
         }
     }
-    @if(!hit.template.ss_dssp.isEmpty) {
+
+    @if(!hit.template.ss_pred.isEmpty) {
```
Comment (zy4) [inline modules/results/src/main/twirl/views/hhpred/hit.scala.html]: this one has to be older

Comment (vikramalva) [reply modules/results/src/main/twirl/views/hhpred/hit.scala.html]: Yes and I just noticed it.

**Pull request merged**